fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/api/resource/conditions/v1/DefaultResourceConditions.java

```diff
@@ -159,6 +168,54 @@ public static ConditionJsonProvider featuresEnabled(FeatureFlag... features) {
 		return ResourceConditionsImpl.featuresEnabled(FEATURES_ENABLED, features);
 	}
 
+	/**
+	 * Creates a condition that returns true if all the passed items are loaded.
+	 *
+	 * @see #registryContains(RegistryKey, Identifier...)
+	 */
+	public static ConditionJsonProvider itemsLoaded(ItemConvertible... items) {
```

Comment:
Maybe `itemsRegistered` since it wraps `registry_contains`? I don't think we usually talk about loading items, except item models and such